dashboard/app/models/course_offering.rb

```diff
@@ -48,6 +48,11 @@ class CourseOffering < ApplicationRecord
   ELEMENTARY_SCHOOL_GRADES = %w[K 1 2 3 4 5].freeze
   MIDDLE_SCHOOL_GRADES = %w[6 7 8].freeze
   HIGH_SCHOOL_GRADES = %w[9 10 11 12].freeze
+  PROFESSIONAL_LEARNING_PROGRAM_PATHS = {
+    'K5 Workshops': 'code.org/professional-development-workshops',
+    '6-12 Workshops': 'code.org/apply',
+  }
+  validates :professional_learning_program, acceptance: {accept: PROFESSIONAL_LEARNING_PROGRAM_PATHS.values, message: "must be one of the professional learning program path. Expected one of: #{PROFESSIONAL_LEARNING_PROGRAM_PATHS.values}. Got:  \"%{value}\"."}
```

Comment:
I'm realizing that once this field is set, you can't go back to 'None':

<img width="665" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/9142121/0c55fd22-a418-45ad-9bef-2af34db9a6aa">

Could you allow `nil` to be an acceptable field?

Unfortunately, this is true with our other categories as well (can't switch Curriculum Type back to 'None', even though we do allow a nil field). We could also split out that work into a separate ticket, since it's an existing problem. Let me know which approach you want to do (fix it now, or split it into another task and tackle the other fields here too).